test/spark/app/src/main/scala/Sonnets.scala

```diff
@@ -41,6 +62,27 @@ object Sonnets {
         )
         target.save(outputPath)
 
+        {
```

Comment:
question: any reason why it needs to be scoped?

Reply:
Just limiting the scope of the variables.